.gitignore

```diff
@@ -58,3 +58,5 @@ scripts/curl.fish
 curl_fuzzer
 curl_fuzzer_seed_corpus.zip
 libstandaloneengine.a
+cmake-build-debug
+.idea
```

Comment:
I don't think we should ignore IDE files for the following reasons:
1) There are just too many editors
2) The committer is responsible for adding these files. Also there are ways to globally ignore some files with git locally

Reply:
1. Not so many as you may think.
2. some files from .idea which contains CLion related setting (e.g. code style) can be added by authors of Curl. If they decide to do that then they can edit the .gitignore
yes, I can exclude files locally with  .git/info/exclude but why not to exclude them for everyone like it made for Eclipse .project and others

Reply:
I think we should not re invent the wheel. To this day, I have never seen a bigger C and C++ open source project ignoring IDE files. Also in the C and C++ development "space" there aren't de-facto IDEs like there are in other languages (Java for example).

Reply:
I've done similar request in the past. It has been rejected, I have been disappointed. Years passed and now I think it's the right decision not to ignore those files at the project level. There is always one more IDE or something else that puts files in the project folder that somebody would want to add to ignore list and nobody want's to manage that.

Reply:
I'm a firm **deny** on adding ignore on anything that is made and handled entirely outside of curl. You can add a global ignore for such files if you want to.